### DISCUSSION TITLE
error with loading gamma-processed data

### DISCUSSION THREAD

**Discussion by AuRevoirzzz** [insarlab/MintPy#1282] [Q&A]
When I load the gamma-processed data, the following error occurred.  I used gamma to process the Lutan-1 data. The data I processed with the latest version of the gamma software was different from the file data structure format recommended by mintpy, and I checked all the data processed with gamma and couldn't find a file corresponding to the command rules. I would like to ask how I can solve this problem.
![image](https://github.com/user-attachments/assets/17bdec75-9051-4c91-8be0-01b989ce7f55)
![image](https://github.com/user-attachments/assets/e8d269b4-3f4d-4831-943f-097be6af90e4)

**Comment by keepfighting666** [comment]
you need to organize data in right form.
